test/unit/CMakeLists.txt

```diff
@@ -600,3 +600,8 @@ create_unit_test(PREFIX lua_tweaks
                            ${LIBYAML_LIBRARIES}
                            ${READLINE_LIBRARIES}
 )
+
+create_unit_test(PREFIX security_gh_63_mpstream_encode_double_null_dereference
+                 SOURCES security_gh_63_mpstream_encode_double_null_dereference.c core_test_utils.c
```

Comment:
Nit: the line is longer than 80 characters.

Reply:
We do not have a line length limit for CMake files in our code style. And we don't have one because CMake files tend to have long lines.